src/cplscheme/CompositionalCouplingScheme.hpp

```diff
@@ -204,6 +204,10 @@ class CompositionalCouplingScheme final : public CouplingScheme {
   /// True if the implicit scheme has converged or no implicit scheme is defined
   bool hasConverged() const final;
 
+  bool requiresSubsteps() const final override;
+
+  std::vector<DataID> implicitDataToReceive() const override final;
```

Comment:
```suggestion
  /// @copydoc cplscheme::CouplingScheme::requiresSubsteps()
  bool requiresSubsteps() const override final;

  /// @copydoc cplscheme::CouplingScheme::implicitDataToReceive()
  std::vector<DataID> implicitDataToReceive() const override final;
```

Do we prefer `final override` or `override final`? In the end it does not really matter, but would be good to do this consistently and maybe even enforce it with the linter.

Reply:
I think `final override` makes visually more sense.
No real point trying to enforce this unless clang-format can deal with this automatically.